ui/components/app/transaction-decoding/components/ui/accreditation/accreditation.component.js

```diff
@@ -45,10 +44,10 @@ const Accreditation = ({ fetchVia, address }) => {
               {fetchVia}
             </Button>,
           ])}
-        </Typography>
-        <Typography variant={TypographyVariant.H7} boxProps={{ margin: 0 }}>
+        </Text>
+        <Text variant={TextVariant.bodySm} as="h6" margin={0}>
```

Comment:
Shouldn't need to reset margin here
```suggestion
        <Text variant={TextVariant.bodySm} as="h6">
```